docs/native-modules-setup.md

```diff
@@ -45,154 +45,64 @@ The steps to create a new native module library project are:
-
-> **Important:** You want *C++/WinRT*, not *C++/CX*. Do **not** choose the C++/CX `Windows Runtime Component (Universal)` project-type:
->
-> ![Wrong Windows Runtime Component (Universal)](assets/native-modules-setup-wrong-cpp-project.png)
->
-> If you don't see the `Windows Runtime Component (C++/WinRT)` project type, go back and install the _C++/WinRT Visual Studio Extension_ under [Development Environment](#development-environment).
-
-1. Set the `Project Name` to `MyLibrary`.
-1. Set the `Location` to the native module directory (typically the same directory with the `ios` and `android` sub-directories).
-1. Set the `Solution Name` to `MyLibrary`.
-1. Click `Create`.
-
-Next you'll be prompted to select the versions of Windows you'll support. This should match the values for React Native Windows, which as of version 0.62, are:
-
-1. Set the `Target version` to `Windows 10, version 1903 (10.0; Build 18362)`.
-1. Set the `Minimum version` to `Windows 10 Fall Creators Update (10.0; Build 16299)`.
-
-You should now have a new `MyLibrary` solution file at `.\MyLibrary\MyLibrary.sln` and a `MyLibrary` project at `.\MyLibrary\MyLibrary\MyLibrary.csproj` for C# or `.\MyLibrary\MyLibrary\MyLibrary.vcxproj` for C++.
-
-Additionally, for C++/WinRT projects, you'll need to change the following:
-
-1. Right click on the project and choose `Manage NuGet Packages...`
-    1. Select version 2.0.200316.3 for Microsoft.Windows.CppWinRT package.
-1. Right-click on the project and choose `Properties`
-    1. Under `Linker > Windows Metadata` set `Generate Windows Metadata` to `Yes`.
-
-Now, before we go any further we'll want to rename the root directory of the Windows native code to `windows` to match the peer `android` and `ios` directories:
+Unfortunately the default library template targets older versions of `react` and `react-native` that Windows supports, so you'll need to upgrade to newer versions of `react` and `react-native` in order to add supprot for `react-native-windows`.
 
-1. Close the solution with `File` > `Close Solution`.
-1. Rename that top `MyLibrary` directory `windows`.
-1. Re-open the solution file at `windows\MyLibrary.sln`.
+> For more information on how NPM dependencies work, see [Specifying dependencies and devDependencies in a package.json file](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file).
 
-> *Optional*: Whether you're creating a new native module from scratch, or adding windows support to an existing ios/android module, if you're using git, you'll want to add Visual Studio-specific entries in your project's `.gitignore`.
-> The simplest way to do this is to get [VisualStudio.gitignore](https://raw.githubusercontent.com/github/gitignore/master/VisualStudio.gitignore) and save it as `windows\.gitignore`.
+You can use the `npm info` command to find the correct versions to use. Let's assume you plan on building against the latest `canary` version of `react-native-windows`.
```

Comment:
Might be better to point to `latest` instead of canary for examples, since Canary has weird version numbers, etc.

Reply:
Each version of the doc points to the version that doc is for. The files under `docs` are for the "next" release, after `latest`, which right now is `canary`. Then the next preview starts I'll change it to `preview`.

Reply:
BTW, is there any way we can "unpublish" `preview` from npm when there's no active preview? Or re-point `preview` to latest once we officially release? It's a little weird when `preview` predates `latest`.

Reply:
We can remove tags if we want to. `react-native` uses `next` instead of `preview`, and kind of does the same thing where `next precedes` `latest` in certain cases (I think there's is honestly more confusing).

Reply:
We can take this offline.